be/src/olap/schema_change.cpp

```diff
@@ -46,12 +46,12 @@ using std::vector;
 namespace doris {
```

Comment:
This file should be rename to alter_tablet_handler.cpp?

Reply:
OK, this modification may refer to many codes, I will push it on a lonely commit.